datanode/sqlstore/accounts.go

```diff
@@ -186,7 +186,7 @@ func (as *Accounts) getAccountFromCache(id entities.AccountID) (entities.Account
 	return entities.Account{}, false
 }
 
-func deterministicIDFromAccount(a *entities.Account) entities.AccountID {
+func DeterministicIDFromAccount(a *entities.Account) entities.AccountID {
```

Comment:
why export this func?

Reply:
For test purpose, so I can craft the account ID for the transfers foreign key.